BeanHub.API/src/main/java/com/bean/api/services/RatingService.java

```diff
@@ -47,7 +47,7 @@ public Rating getRatingById(Long ratingId) {
     }
 
     public List<Recipe> getSortedFeed(@RequestParam(value = "userId") int userId,
```

Comment:
Why do you have Request param here?
There is no request being made to the api here.